### PR TITLE
[15.0][IMP] sign_oca: Redirect at the end of the signature process

### PR DESCRIPTION
**Pull request by victoralmau**
Related to https://github.com/OCA/sign/issues/23

Redirect at the end of the signature process

Please @CarlosRoca13 can you review it?

@Tecnativa TT41745

### CODE REVIEW

File: sign_oca/static/src/components/sign_oca_pdf_portal/sign_oca_pdf_portal.js [hunk 1]
```diff
@@ -39,15 +39,25 @@ odoo.define("sign_oca.document_portal_base", function (require) {
...
+                    route:
+                        "/sign_oca/sign/" +
+                        this.props.signer_id +
+                        "/" +
+                        this.props.access_token,
+                    params: {items: this.info.items},
+                })
+                .then((action) => {
+                    // It is not possible to use do_action(), we change to the corresponding
+                    // URL or redirect.
```
Comment (CarlosRoca13) [inline sign_oca/static/src/components/sign_oca_pdf_portal/sign_oca_pdf_portal.js]:
```suggestion
                    // As we are on frontend env, it is not possible to use do_action(), so we 
                    // redirect to the corresponding URL or reload the page if the action is not
                    // an url.
```

**Review by CarlosRoca13 — APPROVED** [review]
Non blocking comment :smile:

**Review by BernatPForgeFlow — APPROVED** [review]
LGTM!

**Review by etobella — APPROVED** [review]

**Pull request merged**